src/renderer/fonts/font_options.rs

```diff
@@ -13,17 +21,47 @@ pub struct FontOptions {
     pub edging: FontEdging,
 }
 
+impl FontWithFeatures {
+    pub fn parse_family_desc(desc: impl AsRef<str>) -> FontWithFeatures {
```

Comment:
This functionality is complex enough, that I think you need to add a few new unit tests. It should probably also ignore whitespace at the start and end, and between plus and the feature.

Reply:
add some testcases~